models/example/order_calculations.sql

```diff
@@ -0,0 +1,30 @@
+with source1 as (
+
+    select * from {{ ref('customers_upload') }}
+
+),
+
+source2 as (
+    
+    select * from {{ ref('orders_upload') }}
+),
+
+calc as (
+
+    select 
+        source1.id as customer_id, 
+        lag(source2.created_at, 1) [ignore nulls] 
```

Comment:
I think you want to pull from `stg_orders` and `stg_order_items`

your query can look like:
```
select
    items.*,
    orders.email,
    orders.customer_id,
    orders.order_date
from items
inner join order using (order_id)
```

and then do any of the additional modeling that you want for the window functions